.github/workflows/main.yml

```diff
@@ -34,13 +34,11 @@ jobs:
           zenml init --analytics_opt_in False
           python ../smoke_test.py && echo "Finished smoke test successfully" || echo "Smoke test failed"
 
+      # Run script directly
       - name: unit-tests
         run: |
           pip install -e .[all]
-          git init
-          zenml init --analytics_opt_in False
-          python zenml/testing/generate_test_pipelines.py
-          pytest zenml
+          tests/testing.sh
```

Comment:
Nice! Have you tested this out? Should it not be `./tests/testing.sh` ?

Reply:
You are correct, and it should be executed in the `tests` folder based on the script's contents. I changed it to cd into the directory and added a ./  before the script file name. Thanks for catching this one